test/test_fs_util.py

```diff
@@ -8,7 +8,7 @@ class TestChDir(unittest.TestCase):
     @patch("keylime.fs_util.os.path.exists")
     @patch("keylime.fs_util.os.makedirs")
     @patch("keylime.fs_util.os.chdir")
-    def test_ch_dir_present(self, chdir_mock, makedirs_mock, exists_mock):
+    def test_ch_dir_present(self, chdir_mock, makedirs_mock, exists_mock):  # pylint: disable=R
```

Comment:
What refactor complain is producing this code? (just for curiosity)

Reply:
It is [`old-no-self-use`](https://pylint.pycqa.org/en/latest/user_guide/messages/refactor/old-no-self-use.html); as the ID of this error has changed, I couldn't find a way to disable it in both Python 3.6 and 3.10.